packages/lab/src/components/Flow/Node/Node.tsx

```diff
@@ -88,6 +89,12 @@ export const HvFlowNode = ({
   className,
   children,
 }: HvFlowNodeProps<unknown>) => {
+  const { registerNode, unregisterNode } = useNodeMetaRegistry();
+  useEffect(() => {
+    registerNode(id, { title, inputs, outputs });
```

Comment:
I think this should be something like:
`registerNode(id, { label: title, inputs, outputs });`

since the `HvFlowNodeMeta` doesn't have a `title` prop right?

```
export interface HvFlowNodeMeta {
  label: string;
  inputs?: HvFlowNodeInput[];
  outputs?: HvFlowNodeOutput[];
}
```